src/patternfly/components/Toolbar/toolbar.scss

```
@@ -210,6 +212,8 @@ $pf-c-toolbar--inset-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl"
 
   align-items: center;
   margin-right: var(--pf-c-toolbar--spacer);
+  row-gap: var(--pf-c-toolbar__group--RowGap);
+  column-gap: var(--pf-c-toolbar__group--ColumnGap);
```

You could combine these these as:

```suggestion
  gap: var(--pf-c-toolbar__group--RowGap) var(--pf-c-toolbar__group--ColumnGap);
```

We do that for other shorthand properties like `border`,`padding`, `margin`, etc